src/contexts/USDValueProvider.tsx

```diff
@@ -61,7 +61,7 @@ export const USDValueProvider: FC<PropsWithChildren<IInit>> = ({ children }) =>
 
   const getPriceFromJupAPI = useCallback(async (addresses: string[]) => {
     const { data }: { data: JupPriceResponse } = await fetch(
-      `https://price.jup.ag/v4/price?ids=${addresses.join(',')}`,
+      `https://api.jup.ag/price/v2?ids=${addresses.join(',')}`,
```

Comment:
can we use `extraInfo` too ? like the FE, and use the buyPrice and sellPrice

Reply:
let's refrain from requesting more data until we need it?